website/src/components/Messages/MessageTableEntry.tsx

```diff
@@ -108,6 +109,7 @@ export function MessageTableEntry({ message, enabled, highlight, showAuthorBadge
   const { t } = useTranslation(["message"]);
 
   const router = useRouter();
+  const focusElementRef = useFocusElement<HTMLDivElement>();
```

Comment:
We only want to scroll on /messages/${id} page. Can you add another prop `scrollToHightlighted` or something similar? It should be false by default.

Reply:
Maybe better to forward the ref and pass it from MessageTree component

Reply:
done

Reply:
> can you add another prop scrollToHightlighted or something similar?

I added, but not actually currently needed because we don't pass the 'highlighted' prop except in /messages/[id] and /admin/messages/[id] pages